dash/dash.py

```diff
@@ -912,7 +923,85 @@ def dispatch(self):
                 c['id'] == component_registration['id']
             ][0])
 
-        return self.callback_map[target_id]['callback'](*args)
+        output_value = self.callback_map[target_id]['func'](*args)
```

Comment:
The old call to the functions also had `**kwargs`, is that handled somewhere else or just not needed ?

Reply:
I don't think it did, [this](https://github.com/plotly/dash/commit/033084416c334d2dbd4e7888b3e45196abecd112#diff-228c9aefac729977642672aa1416bacaR448) was the last change to that line and it had `*args`